src/main/scala/faces/settings/FacesSettingsJsonFormatV1.scala

```diff
@@ -271,6 +282,30 @@ object ControlledFacesSettingsJsonFormatV1 {
 
 }
 
+object RenderingMethodsJsonVormatV1 {
```

Comment:
Is it nescessary to have an own object? Or could this go into object FacesSettingsJsonFormatV1?

Reply:
Yes it could, I put it outside because it is used by the RandomFacesSettingsJsonFormatV1 and the FacesSettingsJsonFormatV1. Do you think I should put it into one of the two?